app/src/main/java/org/mifos/mobile/ui/transfer_process/TransferProcessViewModel.kt

```diff
@@ -0,0 +1,64 @@
+package org.mifos.mobile.ui.transfer_process
+
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import dagger.hilt.android.lifecycle.HiltViewModel
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.StateFlow
+import kotlinx.coroutines.flow.catch
+import kotlinx.coroutines.launch
+import org.mifos.mobile.models.payload.TransferPayload
+import org.mifos.mobile.repositories.TransferRepository
+import org.mifos.mobile.ui.enums.TransferType
+import org.mifos.mobile.utils.TransferUiState
+import javax.inject.Inject
+
+@HiltViewModel
+class TransferProcessViewModel @Inject constructor(private val transferRepositoryImp: TransferRepository) :
+    ViewModel() {
+
+    private val _transferUiState = MutableStateFlow<TransferUiState>(TransferUiState.Initial)
+    val transferUiState: StateFlow<TransferUiState> get() = _transferUiState
+
+    private var _transferPayload: MutableStateFlow<TransferPayload?> = MutableStateFlow(null)
+    val transferPayload: StateFlow<TransferPayload?> get() = _transferPayload
+
+    private var _transferType: MutableStateFlow<TransferType?> = MutableStateFlow(null)
+    val transferType: StateFlow<TransferType?> get() = _transferType
+    fun makeTransfer(payload: TransferPayload) {
```

Comment:
use proper spacing between lines, and beautify text in Android studio before any commit